corehq/apps/cleanup/management/commands/evaluate_couch_model_for_sql.py

```diff
@@ -196,6 +198,29 @@ def standardize_max_lengths(self):
                 if i < len(max_lengths):
                     params['max_length'] = max_lengths[i]
 
+    def check_unknown_types(self):
```

Comment:
What would you think of also having this function discard fields that were found in docs but don't show up in the couch class definition?

Reply:
yea, that's a good idea - or even just make a note of it in the generated code so the user can decide

Reply:
I can do that in another PR